### ISSUE TITLE
Bug: Configuration system: Validation of arrays prevents setting multiple values easily

### ISSUE THREAD

**Issue by FriedrichWeinmann** [PowershellFrameworkCollective/psframework#54]
By default, only first value will be processed

**Comment by FriedrichWeinmann** [comment]
Proof of error:

# Setting up configuration
```powershell
$AgentNotificationOptionssb = {
    param ([string[]]$input)
    [PSCustomObject]@{success = $true; value = $input}
}

Register-PSFConfigValidation -Name validation.AgentNotifyValidation -ScriptBlock $AgentNotificationOptionssb
Set-PSFConfig -Module dbachecks -Name agent.notifyemail -Validation validation.AgentNotifyValidation -Value @("OnFailure") -Description "blah blah blah"
```

# Try changing to a multi-value setting, then retrieve information
```powershell
Set-PSFConfig -Module dbachecks -Name agent.notifyemail -Value 'foo','bar'
Get-PSFConfig -Module dbachecks -Name agent.notifyemail
```

# Will return only one value
```
    Module: dbachecks


FullName                    Value Description
--------                    ----- -----------
dbachecks.agent.notifyemail foo   blah blah blah
```

**Comment by FriedrichWeinmann** [comment]
# Workaround
Temporary workaround - and one that will break once the fix is done - would be to wrap the input value into an outer array:
```powershell
Set-PSFConfig -Module dbachecks -Name agent.notifyemail -Value @(@('foo','bar'), $null)
Get-PSFConfig -Module dbachecks -Name agent.notifyemail
```
Will show the correct content:
```
    Module: dbachecks


FullName                    Value           Description
--------                    -----           -----------
dbachecks.agent.notifyemail System.Object[] blah blah blah
```

**Comment by FriedrichWeinmann** [comment]
Resolved in 0.9.10.23
